gradlew.bat

```diff
@@ -0,0 +1,100 @@
+@rem
```

Comment:
The generated files have a comment explaining that they were generated at the start. This one doesn't and I wonder if it is supposed to form part of the code base?

Reply:
The wrapper is typically included to allow a user to avoid having to guess the version of gradle used. In this case we aren't using it at any point in the build so it's redundant.